Bearded.UI/Events/EventRouter.cs

```diff
@@ -42,12 +36,60 @@ public static EventPropagationPath FindPropagationPath(IControlParent root, Cont
                     return EventPropagationPath.Empty;
                 }
                 path.Add(parentAsControl);
-                
+
                 parent = parentAsControl.Parent;
             }
 
             path.Reverse();
             return new EventPropagationPath(path.AsReadOnly());
         }
+
+        private static IEnumerable<Control> findPropagationPathUsingTest(
```

Comment:
Why not return the list directly, and avoid the ToList() above?

Reply:
I completely missed that because I was refactoring stuff. Good call!